activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -86,29 +86,11 @@ def associated(*associations)
         @scope
       end
 
-      # Returns a new relation with left outer joins and where clause to identify
-      # missing relations.
-      #
-      # For example, posts that are missing a related author:
-      #
-      #    Post.where.missing(:author)
-      #    # SELECT "posts".* FROM "posts"
-      #    # LEFT OUTER JOIN "authors" ON "authors"."id" = "posts"."author_id"
-      #    # WHERE "authors"."id" IS NULL
-      #
-      # Additionally, multiple relations can be combined. This will return posts
-      # that are missing both an author and any comments:
-      #
-      #    Post.where.missing(:author, :comments)
-      #    # SELECT "posts".* FROM "posts"
-      #    # LEFT OUTER JOIN "authors" ON "authors"."id" = "posts"."author_id"
-      #    # LEFT OUTER JOIN "comments" ON "comments"."post_id" = "posts"."id"
-      #    # WHERE "authors"."id" IS NULL AND "comments"."id" IS NULL
```

Comment:
Did you delete the docs by accident?

Reply:
I did, apologies!  There were commented out methods for different classes I put up there when I was debugging all of this and I accidentally deleted this when I got rid of those.  I'll add them back!

Reply:
Just added this back.